android/app/src/main/res/layout/item_schedule.xml

```diff
@@ -13,16 +13,36 @@
     android:paddingStart="@dimen/padding_extra_large"
     android:paddingTop="@dimen/padding_medium">
 
-    <TextView
-        android:id="@+id/slot_title"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:ellipsize="end"
-        android:paddingBottom="@dimen/padding_small"
-        android:paddingTop="@dimen/padding_small"
-        android:textColor="@color/black"
-        android:textSize="@dimen/text_size_extra_large"
-        tools:text="Slot Title" />
+    <RelativeLayout
```

Comment:
RelativeLayout is expensive, this can also be done using a LinearLayout. Please use that instead. Thanks 😄

Reply:
@iamareebjamal Oh ok will do.
Thanks :)

Reply:
Changed it to LinearLayout with weights.
Please review :)